docs/autonomy.md

```diff
@@ -423,7 +423,7 @@ autonomy fetch --local --alias oracle valory/oracle:0.1.0
 ```bash
 Usage: autonomy run [OPTIONS]
 
-  Run the agent.
+  Run the agent. Works for local agents as well as docker compose deployments, but not for kubernetes deployments.
```

Comment:
Yes, I didn't know what to call agents that are not part of a docker compose or k8 deployment. Basically an agent that you have locally in your system. Since we are now docker first I can delete it but maybe worths mentioning.

Reply:
They are all local. We have two ways of running: docker compose or kubernetes. So I am not sure what you are getting at.